src/biokbase/narrative/tests/test_jobmanager.py

```diff
@@ -317,29 +331,6 @@ def test_cancel_job__job_already_finished(self):
                 canceled_jobs,
             )
 
-    @mock.patch("biokbase.narrative.jobs.jobmanager.clients.get", get_mock_client)
-    def test_cancel_job__run_ee2_cancel_job(self):
-        """cancel a job that runs cancel_job on ee2"""
-        self.assertIn(JOB_RUNNING, self.jm._running_jobs)
-        self.assertFalse(self.jm.get_job(JOB_RUNNING).was_terminal)
-        # set the job refresh to 1
-        self.jm._running_jobs[JOB_RUNNING]["refresh"] = 1
-
-        def check_state(arg):
-            self.assertEqual(self.jm._running_jobs[arg["job_id"]]["refresh"], 0)
-            self.assertEqual(self.jm._running_jobs[arg["job_id"]]["canceling"], True)
-
-        # patch MockClients.cancel_job so we can test the input
-        with mock.patch.object(
-            MockClients,
-            "cancel_job",
-            mock.Mock(return_value={}, side_effect=check_state),
-        ) as mock_cancel_job:
-            self.jm.cancel_job(JOB_RUNNING)
-            mock_cancel_job.assert_called_once_with({"job_id": JOB_RUNNING})
-            self.assertNotIn("canceling", self.jm._running_jobs[JOB_RUNNING])
-            self.assertEqual(self.jm._running_jobs[JOB_RUNNING]["refresh"], 1)
-
```

Comment:
There's already an equivalent test for `jm.cancel_jobs` (plural) that essentially tests same thing! So it's okay to just let this go